### PR TITLE
chore: Migrate News section to Firestore

### PR DESCRIPTION
**Pull request by harshtaliwal30**
Migrate news section database to cloud firestore.

### Files Changed
- store/news.js
- store/index.js
- plugins/firebase.js
- components/news/Admin.vue
- package.json
- yarn.lock

### CODE REVIEW

File: plugins/firebase.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ if (!firebase.apps.length) {
 }
 
 export const db = firebase.database()
+export const firestoreDb = firebase.firestore()
```
Comment (HashirGJ8842) [inline plugins/firebase.js]: Rename this variable to DB

File: components/news/Admin.vue [hunk 2]
```diff
@@ -164,7 +168,7 @@ export default {
       this.$emit('update:selectedToEdit', null)
     },
     deleteNewsItem() {
-      this.deleteNews(this.selectedToDelete['.key'])
+      this.deleteNews(this.selectedToDelete)
```
Comment (HashirGJ8842) [inline components/news/Admin.vue]:
```suggestion
      this.deleteNews(this.selectedToDelete.id)
```

then pass the `id` in store/news.js

**Review by HashirGJ8842 — COMMENTED** [review]
Nice work, update this branch to resolve conflicts

**Comment by harshtaliwal30** [comment]
Updated the pull request with the required changes.

**Review by HashirGJ8842 — APPROVED** [review]

**Pull request merged**